vlasiator.cpp

```diff
@@ -881,9 +882,9 @@ int main(int argn,char* args[]) {
       
       // Apply boundary conditions
       if (P::propagateVlasovTranslation || P::propagateVlasovAcceleration ) {
-         phiprof::start("Update system boundaries (Vlasov post-translation)");
+         phiprof::start("Update system sysBoundaries (Vlasov post-translation)");
          sysBoundaries.applySysBoundaryVlasovConditions(mpiGrid, P::t+0.5*P::dt, false);
-         phiprof::stop("Update system boundaries (Vlasov post-translation)");
+         phiprof::stop("Update system sysBoundaries (Vlasov post-translation)");
          addTimedBarrier("barrier-boundary-conditions");
```

Comment:
"system sysBoundaries" is a bit of a repeat now, either "system boundaries" or "sysBoundaries" I'd say.

Reply:
I think I messed up with my other branch where `sysboundary` are renamed to `boundary`. I will revert the changes here.

Reply:
ok